server/src/routes/api/v1/indexRouter.js

```diff
@@ -0,0 +1,16 @@
+import express from "express"
+
+import { Trail } from "../../../models/index.js"
+
+const indexRouter = new express.Router()
```

Comment:
The name of our router should reflect the resource it is interacting with, in this case `trails`